app/src/views/disk-view.js

```diff
@@ -46,7 +47,9 @@ export default class DiskView {
   }
 
   _handleChanges(changes) {
-    if (changes.hasOwnProperty('disk') && changes.disk.hasOwnProperty('level') || changes.currentGame === "disk") {
+    if (changes.currentGame === GAMES.HANDSHAKE || // Reset on startup
```

Comment:
This adds a dependency on the game sequence and will break if we change it. Instead, just call reset disks from the constructor.